### ISSUE TITLE
Slow synchronization

### ISSUE THREAD

**Issue by dg123lzk** [allada/bsc-archive-snapshot#52]
Hello, I downloaded the snapshot yesterday. The synchronization speed was very fast at first, but it was slow after a period of time. The machine was configured according to the recommended configuration. 'exec erigon --chain bsc --snapshots=true --db.pagesize=16k --datadir=/erigon/data/bsc --txpool.disable --http -- HTTP. Addr "0.0.0.0" - HTTP. API "web3 eth,.net, admin, debug, txpool" '

It was synchronized in the same way before, and there was no slow synchronization problem before

`Mar 15 03:20:02 ip-172-31-2-38 start_erigon_service.sh[1591536]: [INFO] [03-15|03:20:02.623] [7/16 Execution] Executed blocks         number=25946797 blk/s=9.3 tx/s=1808.8 Mgas/s=229.6 gasState=0.17 batch=192.9MB alloc=3.7GB sys=5.8GB
Mar 15 03:20:22 ip-172-31-2-38 start_erigon_service.sh[1591536]: [INFO] [03-15|03:20:22.607] [7/16 Execution] Executed blocks         number=25947019 blk/s=11.1 tx/s=1875.1 Mgas/s=239.0 gasState=0.18 batch=202.6MB alloc=3.0GB sys=5.8GB
Mar 15 03:20:42 ip-172-31-2-38 start_erigon_service.sh[1591536]: [INFO] [03-15|03:20:42.618] [7/16 Execution] Executed blocks         number=25947254 blk/s=11.7 tx/s=2037.2 Mgas/s=236.2 gasState=0.19 batch=212.3MB alloc=4.5GB sys=5.8GB
Mar 15 03:21:02 ip-172-31-2-38 start_erigon_service.sh[1591536]: [INFO] [03-15|03:21:02.654] [7/16 Execution] Executed blocks         number=25947495 blk/s=12.0 tx/s=1922.4 Mgas/s=244.0 gasState=0.20 batch=222.2MB alloc=3.4GB sys=5.8GB
Mar 15 03:21:22 ip-172-31-2-38 start_erigon_service.sh[1591536]: [INFO] [03-15|03:21:22.596] [7/16 Execution] Executed blocks         number=25947731 blk/s=11.8 tx/s=2021.7 Mgas/s=248.1 gasState=0.21 batch=232.5MB alloc=2.8GB sys=5.9GB
Mar 15 03:21:42 ip-172-31-2-38 start_erigon_service.sh[1591536]: [INFO] [03-15|03:21:42.645] [7/16 Execution] Executed blocks         number=25947873 blk/s=7.1 tx/s=1375.5 Mgas/s=177.0 gasState=0.21 batch=240.1MB alloc=4.1GB sys=5.9GB
Mar 15 03:21:49 ip-172-31-2-38 start_erigon_service.sh[1591536]: [INFO] [03-15|03:21:49.324] [p2p] GoodPeers                          eth66=14 eth67=5
Mar 15 03:22:02 ip-172-31-2-38 start_erigon_service.sh[1591536]: [INFO] [03-15|03:22:02.624] [7/16 Execution] Executed blocks         number=25948101 blk/s=11.4 tx/s=1971.3 Mgas/s=243.6 gasState=0.22 batch=250.7MB alloc=3.1GB sys=5.9GB
Mar 15 03:22:12 ip-172-31-2-38 start_erigon_service.sh[1591536]: [INFO] [03-15|03:22:12.827] Committed State                          gas reached=124787624522 gasTarget=549755813888
Mar 15 03:23:01 ip-172-31-2-38 start_erigon_service.sh[1591536]: [INFO] [03-15|03:23:01.944] [] ETL [2/2] Loading                     into=PlainState current key=1f474b74...
Mar 15 03:23:27 ip-172-31-2-38 start_erigon_service.sh[1591536]: [INFO] [03-15|03:23:27.819] [] ETL [2/2] Loading                     into=PlainState current key=2af55ab3...
Mar 15 03:23:49 ip-172-31-2-38 start_erigon_service.sh[1591536]: [INFO] [03-15|03:23:49.324] [p2p] GoodPeers                          eth66=14 eth67=6
Mar 15 03:24:01 ip-172-31-2-38 start_erigon_service.sh[1591536]: [INFO] [03-15|03:24:01.417] [] ETL [2/2] Loading                     into=PlainState current key=3d24c455...
Mar 15 03:24:34 ip-172-31-2-38 start_erigon_service.sh[1591536]: [INFO] [03-15|03:24:34.187] [] ETL [2/2] Loading                     into=PlainState current key=55d39832...`

**Comment by allada** [comment]
Yes syncing can be quite slow. You can try adding more hardware, usually additional SSDs help.
